### PR TITLE
New Style Errors for Pubmed

### PR DESCRIPTION
**Pull request by nikicc**

### CODE REVIEW

File: orangecontrib/text/widgets/ownyt.py [hunk 1]
```diff
@@ -52,6 +52,16 @@ class OWNYT(OWWidget):
     includes_web_url = Setting(False)
     includes_word_count = Setting(False)
 
+    class Warning(OWWidget.Warning):
+        no_query = Msg('Please enter a query before attempting to fetch results.')
+        no_text_fields = Msg('You must select at least one text field.')
+        start_after_end = Msg('The start date is greater than the end date.')
+        invalid_date_format = Msg('Invalid date interval endpoint format (must be YYYY/MM/DD)')
+        to_early_start = Msg('There are no records before the year {}. '
```
Comment (kernc) [inline orangecontrib/text/widgets/ownyt.py]: It's _too_ early.

Comment (kernc) [inline orangecontrib/text/widgets/ownyt.py]: But this warning isn't used anywhere?

**Comment by ajdapretnar** [comment]
This seems ok, but I've got no idea how to replicate the API error. 😮

**Comment by nikicc** [comment]
@ajdapretnar I would just put this into try except block in `pubmed.py` (line 246). It should raise an error with probability 0.5 😄 
```py
import random
if random.random() < 0.5:
    raise IOError('Random API error!')
```

**Pull request merged**